modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesKernelBoot.java

```diff
@@ -301,6 +318,24 @@ public static void startAndesStores() throws Exception {
                              .setVirtualHostConfigSynchronizer(_VirtualHostConfigSynchronizer);
     }
 
+    /**
+     * Initialize mode of cluster event synchronization depending on configurations.
+     */
+    private static void initClusterEventSynchronization() {
+
+        String clusterEventSyncMode = AndesConfigurationManager.readValue(AndesConfiguration.CLUSTER_EVENT_SYNC_MODE);
```

Comment:
this could be backed by a enum ( check the andes configuration management for similar example)